frontend/src/tests/pages/PageNotFound.test.js

```diff
@@ -0,0 +1,30 @@
+import { render } from "@testing-library/react";
+import PageNotFound from "main/pages/PageNotFound";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { MemoryRouter } from "react-router-dom";
+
+import { apiCurrentUserFixtures }  from "fixtures/currentUserFixtures";
+import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+import axios from "axios";
+import AxiosMockAdapter from "axios-mock-adapter";
+
+describe("PageNotFound tests", () => {
+
+    const axiosMock =new AxiosMockAdapter(axios);
+    axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.userOnly);
+    axiosMock.onGet("/api/systemInfo").reply(200, systemInfoFixtures.showingNeither);
+
+    const queryClient = new QueryClient();
+    test("renders without crashing", () => {
+        render(
+            <QueryClientProvider client={queryClient}>
+                <MemoryRouter>
+                    <PageNotFound />
+                </MemoryRouter>Cheryl
```

Comment:
Should the word `Cheryl` be here, or is this a typo?

Reply:
This was a typo, will go ahead and fix that thank you for pointing it out!